tools/ui-manta-installer.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+#
+# Copyright 2024 MNX Cloud, Inc.
+#
+
+set -o errexit
+set -o pipefail
+
+while getopts "b:n" options; do
+   case $options in
+      b ) BRANCH="${OPTARG}";;
+      n ) NOT_REALLY=1;;
+      * ) usage ;;
+   esac
+done
+
+MANTA_URL=https://us-central.manta.mnx.io
+LATEST_URI="/Joyent_Dev/public/builds/smartos-ui/${BRANCH:=master}-latest"
```

Comment:
We don't do that for anything else, so I don't think it will be an issue.

There were at times that Joyent did big purges of branch builds, but it wasn't a regular thing.